pkg/constants/constants.go

```diff
@@ -206,6 +206,11 @@ const (
 	HTTPServerSmiVersionPath = "/smi/version"
 )
 
+// Windows Specific containers
+const (
+	WindowsNanoserverDockerImage = "mcr.microsoft.com/windows/nanoserver/insider:10.0.20348.1"
```

Comment:
Should this be configurable?

Reply:
What is the scenario where someone would want to change it? Any WS2022 image should work here

Reply:
This seems to be used only in tests, so move this constant to `tests/framework`.

Reply:
+1, there are two constants.go files and probably my idea opened the non-test one first

Reply:
moved it into the correct constants file